lambda/simplif.ml

```diff
@@ -821,7 +847,14 @@ let simplify_local_functions lam =
       | Llet (_, _, id, _, cont) when Hashtbl.mem static_id id ->
           rewrite cont
       | Lapply {ap_func = Lvar id; ap_args; _} when Hashtbl.mem static_id id ->
-          Lstaticraise (Hashtbl.find static_id id, List.map rewrite ap_args)
+          begin
+            let st = Hashtbl.find static_id id in
+            let slot = Hashtbl.find slots id in
+            match exact_application slot.func ap_args with
```

Comment:
Purely cosmetic suggestion again: it seems to be a more common style to use `begin match ... end` rather than enclosing the whole action wirh `begin...end` (and it avoids some extra indentation).

Reply:
Done.